test/functional/services/dashboard/add_panel.js

```diff
@@ -132,7 +154,7 @@ export function DashboardAddPanelProvider({ getService, getPageObjects }) {
       log.debug(`DashboardAddPanel.addVisualization(${vizName})`);
       await this.ensureAddPanelIsShowing();
       await this.filterEmbeddableNames(`"${vizName.replace('-', ' ')}"`);
-      await find.clickByPartialLinkText(vizName);
+      await testSubjects.click(`addPanel${vizName.split(' ').join('-')}`);
```

Comment:
Since you added this in, do you still need to have the logic that changes all the spaces to dashes in the callers?

Reply:
Not sure what you mean here. Could you explain a little more?